modules/govuk/templates/ckan/nginx.conf.erb

```diff
@@ -71,6 +71,11 @@ location /api/3<%= path %> {
 }
 <% end %>
 
+# Rewrite publisher to organization for CKAN 2.9 in case it has been bookmarked
+location /publisher {
+  rewrite ^(.*)/publisher(/.*)?$ $1/organization$2 last;
```

Comment:
If this is for redirecting people who may have bookmarked the old location, consider using 301 Moved Permanently (`permanent`).

Reply:
I might give that a try